core/Symbols.cc

```diff
@@ -112,6 +112,8 @@ TypePtr ClassOrModule::selfType(const GlobalState &gs) const {
     }
 }
 
+// ClassOrModule::resultType is computed by unsafeComputeExternalType,
```

Comment:
Should this say `ClassOrModule:: externalType is computed by...`?

Reply:
I think it should say `resultType`. `externalType` is computed by this method specifically, but the implementation of this method relies on having had `this->resultType` set already, and that field gets set by the mentioned method.